pallets/author-filter/src/lib.rs

```diff
@@ -67,6 +67,9 @@ pub mod pallet {
 	impl<T: Config> author_inherent::CanAuthor<T::AccountId> for Pallet<T> {
 		fn can_author(account: &T::AccountId) -> bool {
 			let mut staked: Vec<T::AccountId> = stake::Module::<T>::validators();
+			if staked.len() == 1 {
```

Comment:
Is goal here is to choose only 1 eligible author per block from the set of validators? Right now, this condition allows authorship when the passed in account is the only account in the set of validators.

I suggest changing `EligibleRatio` to `EligibleCount: u32` and setting it to 1. Then, changing the logic below to `let mut eligible = Vec::with_capacity(EligibleCount::<T>::get() as usize)`. We need a comment somewhere that `EligibleCount` should always be less than `MaxValidators`.